### ISSUE TITLE
Wheels for Mac OS and Windows

### ISSUE THREAD

**Issue by shwina** [NVIDIA/NVTX#81]
Currently, we only publish wheels for Linux/x86 on PyPI: https://pypi.org/project/nvtx/#files. On any other OS/arch, pip installs fall back to source builds, which are problematic because users need to configure their compilers to find the NVTX C headers. This is often a pain point, as evidenced by issues like #80.

We should provide pre-built wheels for other OS/arch.

**Comment by Meerkov** [comment]
Yes please, you're a $3 trillion company, can we have windows support :)

**Comment by evanramos-nvidia** [comment]
Implemented on the dev branch in commit 5ed4270312e09ee2234d08102a542607845088e1

**Comment by nitinmukesh** [comment]
> Yes please, you're a $3 trillion company, 

Not anymore.

> can we have windows support :)

This worked for me 
pip install git+https://github.com/NVIDIA/NVTX.git@dev#subdirectory=python
